src/main/java/frc/robot/RobotContainer.java

```diff
@@ -30,6 +34,12 @@ public static RobotContainer getInstance() {
     }
 
     private void configureDefaultCommands() {
+        if (rb.getAsBoolean()) {
+            Leds.getINSTANCE().setYellow();
+        } else if (lb.getAsBoolean()) {
+            Leds.getINSTANCE().setPurple();
+        }
+
```

Comment:
1. Redundant newline added at the end of the function.
2. You should save `Leds.getINSTANCE()` to a `final` variable and not call it each time.
3. Notice the function right below, `configureButtonBindings()`. Perhaps this code should live there?
4. You shouldn't declare command bindings that way. Read [here](https://docs.wpilib.org/en/stable/docs/software/commandbased/binding-commands-to-triggers.html) for more information.